m4opt/missions/_lsst/lsst_camera.py

```diff
@@ -0,0 +1,82 @@
+"""
+Detector positions and LSST Camera geometry are extracted from:
+
+    - LSSTCam: https://github.com/lsst/obs_lsst/blob/main/policy/lsstCamSim.yaml
+    - Science detectors are extracted, excluding wavefront and guide sensors.
+    - Offsets and pixel sizes are provided in millimeters (mm).
+    - The extracted data is saved in CSV formats for or processing.
+"""
+
+from importlib import resources
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+from regions import PolygonSkyRegion, RectangleSkyRegion, Regions
+
+from . import data
+
+
+class LSSTfieldOfView:
```

Comment:
Not sure why this is a class.

Reply:
I can also  do it like this : http://github.com/weizmannk/EarthOrbitPlan/blob/main/lsst_fov/lsst_geometry_camera.py